german/the-black-hack.md

```diff
@@ -214,7 +214,7 @@ Wenn Charaktere Waffen verwenden, die nicht bei ihrer Klasse aufgelistet stehen,
 
 ## Zufallsbegegnungen
 
-Ein SL kann alle 15 Minuten in der realen Welt (ihr *haltet* das nach, oder?) einen W4 würfeln. Bei einem Ergebnis von 1-2 taucht innerhalb der nächsten **Minute** (Runde) ein zufällig ermitteltes Wesen oder eine Ablenkung auf.
+Ein SL kann alle 15 Minuten in der realen Welt (ihr *achtet darauf*, oder?) einen W4 würfeln. Bei einem Ergebnis von 1-2 taucht innerhalb der nächsten **Minute** (Runde) ein zufällig ermitteltes Wesen oder eine Ablenkung auf.
```

Comment:
Wobei halt eigentlich nur der SL auf die Zeit achten sollte. Sonst fangen die Spieler damit an und fragen wo denn ihre Zufallsbegegnung bleibt.^^ Daher auch das direkte an den SL wenden, schätze ich. Aber klar sprachlich ist das kein guter Stil, dann hätte es besser direkt mit "Du als SL solltest alle 15..." Aber dann müsste alles entsprechend angepasst werden.
Vielleicht den Teil in den Klammern einfach komplett ändern zu einem "(also immer schön auf die Zeit achten)" ?